README.md

```diff
@@ -49,7 +49,7 @@ You can configure additional options using `cloudflareDev: { }` in `nitro.config
 - Install the latest LTS version of [Node.js](https://nodejs.org/en/)
 - Enable [Corepack](https://github.com/nodejs/corepack) using `corepack enable`
 - Install dependencies using `pnpm install`
-- Build in stub mode using `pnpm build --stub`
+- Prepare the build in stub mode using `pnpm dev:prepare`
```

Comment:
Please, i rather to make dev experience aligned with `unbuild` command.

Reply:
As you wish... I still believe the `dev:prepare` command is better in term of DX, people does not need to know what's dark magic is happening under the hood.

Reply:
I wish no `dev:prepared` was needed at all  and `build --stub` is not dark magic. At least it is more readable than tracing via another script to find out what it calls.